controller/source/rocksdb_server/server.cpp

```diff
@@ -0,0 +1,95 @@
+#include <iostream>
+#include <string>
+#include <span>
+#include <vector>
+#include <memory>
+#include <boost/asio.hpp>
+
+#include "rocksdb_proxy.hpp"
+
+using boost::asio::ip::tcp;
+
+class session : public std::enable_shared_from_this<session> {
```

Comment:
Add a comment about what the session object refers to (because it's a generic term)
@ertugrulaypek

Reply:
We might need a similar concept for redis client so in a next version we might need to generlize the `session` term.

Reply:
added docstrings to `session`, `rocksdb_proxy` and `rocksdb_server` classes.